internal/policy/policy.go

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright 2020 The Flux authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package policy
+
+type Interface interface {
```

Comment:
I'd name this something that describes what it does. (The interface in go's sort package defines the single requirement for `sort`, which doesn't typically need to be named outside of its own package -- so it can be forgiven for being named `sort.Interface`.)

Reply:
Fine, then I'll name it `Policer` (I had to check if that word exists and was pleasantly surprised it does).